docs/source/kernel-spark-standalone.md

```diff
@@ -0,0 +1,142 @@
+## Spark Standalone
+
+Jupyter Enterprise Gateway extends Jupyter Kernel Gateway which means that by installing kernels in Enterprise Gateway and using the vanila kernelspecs created during installation you will have your kernels running in client mode with drivers running on the same host as Enterprise Gateway. 
+
+Having said that, even if you are not leveraging the full distributed capabilities of Jupyter Enterprise Gateway, client mode can still help mitigate resource starvation by enabling a pseudo-distributed mode, where kernels are started in different nodes of the cluster utilizing a round-robin algorithm. In this case, you can still experience bottlenecks on a given node that receives requests to start "large" kernels, but otherwise, you will be better off compared to when all kernels are started on a single node or as local processes, which is the default for vanilla Jupyter Notebook.
+
+The pseudo-distributed capabilities are currently supported in Spark Standalone and require the following environment variables to be set:
+
+* SPARK_HOME: Must point to the Apache Spark installation path
+
+```
+SPARK_HOME:/usr/hdp/current/spark2-client                            #For HDP distribution
+```
+
+* EG_REMOTE_HOSTS must be set to a comma-separated set of FQDN hosts indicating the hosts available for running kernels. (This can be specified via the command line as well: `--EnterpriseGatewayApp.remote_hosts`)
+
+```
+EG_REMOTE_HOSTS=elyra-node-1.fyre.ibm.com,elyra-node-2.fyre.ibm.com,elyra-node-3.fyre.ibm.com,elyra-node-4.fyre.ibm.com,elyra-node-5.fyre.ibm.com
+```
+
+### Configuring Kernels for Spark Standalone
+
+We will use some `yarn_client` specs for Spark Standalone. Follow [#681](https://github.com/jupyter/enterprise_gateway/issues/681) that will ensure Spark Standalone gets its own kernelspec.
```

Comment:
#681 is a duplicate of #346.  I'd prefer we not reference an open issue in the docs but instead say something something like, _Although Enterprise Gateway does not currently provide sample kernelspecs for Spark standalone, here are the steps necessary to convert a `yarn_client` kernelspec to standalone._

(We view the kernelspecs we do provide as samples since, in nearly all cases, tweaks and customizations for the target environment will be required.)